docs/src/testing/running-tests.md

```diff
@@ -24,9 +26,11 @@ Collected 2 test(s) from package_name package
 Running 2 test(s) from src/
 [PASS] package_name::tests::calling
 [PASS] package_name::tests::calling_another
-Tests: 2 passed, 0 failed, 0 skipped, 0 ignored, 1 filtered out
+Tests: 2 passed, 0 failed, 0 skipped, 0 ignored, other filtered out
 ```
 
+When running with a filter, `other filtered out` is displayed in the summary.
```

Comment:
I think that instead of explaining cases in which it is displayed we should briefly explain optimized compilation logic here - maybe as a separate section. Just say something along the lines of "if you filter out tests using just a name, not paths (`::`), we will only compile these tests resulting in faster compilation`.

Reply:
Actually I've reworded it and added a note, should draw user's attention more 👍 .